pkgs/top-level/all-packages.nix

```diff
@@ -12236,6 +12246,14 @@ in
   conkeror-unwrapped = callPackage ../applications/networking/browsers/conkeror { };
   conkeror = self.wrapFirefox conkeror-unwrapped { };
 
+  cpp_ethereum = callPackage ../applications/misc/ethereum/cpp {
```

Comment:
Perhaps this should be called "webtree" instead? I don't understand the difference in naming conventions but the "cpp-ethereum" repo is empty besides some git modules.